apps/web/src/pages/templates/workflow/workflow/node-types/WorkflowNodeActions.tsx

```diff
@@ -110,17 +119,22 @@ export const WorkflowNodeActions = ({
   }
 
   const isDelayedStep = DELAYED_STEPS.includes(channelType);
+  const conditionsAction = isReadOnly
+    ? VARIANT_TYPE_TO_VIEW_CONDITIONS[nodeType]
+    : conditionsCount > 0
+    ? VARIANT_TYPE_TO_EDIT_CONDITIONS[nodeType]
+    : VARIANT_TYPE_TO_ADD_CONDITIONS[nodeType];
```

Comment:
in the production env show the tooltip text for the conditions like: `View conditions`